Library/Formula/watch.rb

```diff
@@ -12,15 +18,47 @@ class Watch < Formula
     sha1 "a7c559378bc74cd30d00f962e63d6ee5c705aea1" => :mountain_lion
   end
 
+  depends_on "gettext" => :build
```

Comment:
@DomT4 `watch` needs `autopoint` from `gettext` package when running `./autogen.sh`, but not when configured with `--disable-nls` flag:
```
autopoint --force $AP_OPTS
if ! grep -q datarootdir po/Makefile.in.in; then
        echo autopoint does not honor dataroot variable, patching.
        sed -i -e 's/^datadir *=\(.*\)/datarootdir = @datarootdir@\
datadir = @datadir@/g' po/Makefile.in.in
fi
```